### ISSUE TITLE
Can i use this to build desktop and mobile in parallel?

### ISSUE THREAD

**Issue by Ajeetlakhani** [mysticatea/npm-run-all#186]
Hi,

I want to build parallel builds for mobile and desktop react application.

Can i use this to create build parallely and would it be standard way of doing it?

**Comment by TehShrike** [comment]
Yup.  I use it for that.  As long as your build tool doesn't have any problems with it you're good to go
